Hazel/src/Platform/OpenGL/OpenGLContext.cpp

```diff
@@ -24,6 +24,15 @@ namespace Hazel {
 		HZ_CORE_INFO("  Renderer: {0}", glGetString(GL_RENDERER));
 		HZ_CORE_INFO("  Version: {0}", glGetString(GL_VERSION));
 
+		#ifdef HZ_ENABLE_ASSERTS
+		int versionMajor;
+		int versionMinor;
+		glGetIntegerv(GL_MAJOR_VERSION, &versionMajor);
+		glGetIntegerv(GL_MINOR_VERSION, &versionMinor);
+
+		HZ_CORE_ASSERT(versionMajor >= 4, "Hazel requires at least OpenGL version 4.5!");
+		HZ_CORE_ASSERT(versionMinor >= 5, "Hazel requires at least OpenGL version 4.5!");
```

Comment:
Once again, rather make these a single assert.

Reply:
Oh. Forgot this.